### ISSUE TITLE
Vertical Nav (Mixed): Label, state and menu/submenu items related info is not communicated appropriately

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4624]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, Role, Value.

**Test Environment:**
- Test URL: https://clarity.design/documentation/vertical-nav/basic-structure/project-pokemon
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to the mixed navigation heading level 5 and navigate through the vertical nav group provided under it for Project Pokemon.

**Expected result:**
Nav groups label should be same as the visual label and it should be announced by the screen reader.

**Actual result:**
All the collapsible groups in the vertical Nav are announced as &quot;Toggle vertical navigation, button&quot; followed by the &quot;expand/collapsed&quot; state. User is not notified about the number of items in the vertical nav, nor user is notified about the submenu items appropriately.

**Proposed solution:**
Aria-label attribute should be unique for each group. Toggle word should not be included in the label as aria-expanded attribute conveys that semantics. So a group label can be &quot;Electric, button, expanded&quot;. Instead of using group in the label, please use aria-haspopup attribute along with &lt;ul&gt; and &lt;li&gt; tags to identify menu and it&#039;s submenu items properly with additional info like number of items in the menu.

**Reference:**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

**Comment by gkhadK** [comment]
SImilar issue is observed in wizard navigation list clr-wizard-stepnav-list where user cannot differentiate between selected and non selected list items,

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
